.github/workflows/flutter-deploy.yml

```diff
@@ -91,6 +95,11 @@ jobs:
           qbs-maven-username: ${{ secrets.QBS_DEV_MAVEN_USERNAME }}
           qbs-maven-password: ${{ secrets.QBS_DEV_MAVEN_PASSWORD }}
           upload-key: ${{ secrets.UPLOAD_KEY }}
+          obfuscate: ${{ inputs.obfuscate }}
+          sentry-url: ${{ secrets.SENTRY_URL }}
+          sentry-org: ${{ secrets.SENTRY_ORG }}
+          sentry-project: ${{ secrets.SENTRY_PROJECT }}
+          sentry-auth-token: ${{ secrets.SENTRY_AUTH_TOKEN }}  
```

Comment:
These are project secrets, correct? Not org ones? I can't remember what happens if a secret is not present in this situation. I think it may cause an error in the workflow, and that would affect most of our projects. Please test and if it works in a repo with no SENTRY_URL, SENTRY_ORG...., it's fine. If not, make these optional inputs and the repo's workflow can pass it on their own

Reply:
Same goes for the same thing in the rest of the PR

Reply:
all except SENTRY_PROJECT are org secrets and the the action to upload debug symbol does not run if SENTRY_PROJECT is not defined.